sdk/java-sdk/src/main/scala/kalix/javasdk/impl/timer/TimerSchedulerImpl.scala

```diff
@@ -44,12 +43,18 @@ private[kalix] final class TimerSchedulerImpl(messageCodec: MessageCodec, system
 
     val timerServiceClient = GrpcClients(system).getProxyGrpcClient(classOf[TimerService])
 
-    val deferredCallImpl = deferredCall.asInstanceOf[DeferredCallImpl[I, O]]
-    val call =
-      Call(
-        deferredCallImpl.fullServiceName,
-        deferredCallImpl.methodName,
-        Some(messageCodec.encodeScala(deferredCall.message())))
+    val call = deferredCall match {
+      case deferredCallImpl: DeferredCallImpl[I, O] =>
```

Comment:
Shall we rename `DeferredCallImpl` to `GrpcDeferredCall`? 

And `RestDeferredCallImpl` to `RestDeferredCall`?

Now that we have two distinct implementations, we don't need to suffix it with `Impl` anymore.

Reply:
Yup, makes sense, tks